libs/platform/core/src/services/hydration/default-hydration.service.ts

```diff
@@ -57,6 +59,11 @@ export class DefaultHydrationService implements HydrationService, OnDestroy {
         return;
       }
 
+      //need to hydrate elements with dynamic visibility rule on load
```

Comment:
actually, we could come up with an idea, that composition that knows, that has dynamic visibility, can just setup hydration for window:onload. No need to pollute hydratable durective then.